frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/columns/simple_columns_block_component.dart

```diff
@@ -194,6 +200,21 @@ class ColumnsBlockComponentState extends State<ColumnsBlockComponent>
     }
   }
 
+  void listenForSizeChanged() {
```

Comment:
Is it an infinite loop to listen for height changes? Maybe we can use `SizeChangedLayoutNotifier` to implement similar logic.